src/crimson/osd/ops_executer.cc

```diff
@@ -566,6 +566,15 @@ OpsExecuter::execute_op(OSDOp& osd_op)
     return do_write_op([&osd_op] (auto& backend, auto& os, auto& txn) {
       return backend.omap_remove_range(os, osd_op, txn);
     }, true);
+  case CEPH_OSD_OP_OMAPRMKEYS:
+#if 0
```

Comment:
Not sure I understand the question. Are you asking why am I using "#if 0"? If so then that is because the method `supports_omap()` is not yet implemented.

Reply:
if that's the case better add a `TODO` accompanied with explanation. a naked `#if 0` is just confusing.

Reply:
Well, this actually follows a convention we already established several lines above. I'm not arguing the convention is good; just pointing out it's already there.

Reply:
I replaced the `#if 0` with a comment. Will make a follow up PR changing the rest of the lines as well.